ursh/indico_ursh/templates/ursh_shortener_page.html

```diff
@@ -0,0 +1,35 @@
+{% extends 'layout/base.html' %}
+
+{% block page_class %}fixed-width-standalone-text-page{% endblock %}
+
+{% block title -%}
+    {% trans %}URL Shortener{% endtrans %}
+{%- endblock %}
+
+{% block content -%}
+    <p>
+        {% trans -%}
+            In this page, you can generate short links for any Indico URL.
+            Just enter the URL below and click "Shorten".
+        {%- endtrans %}
+    </p>
+    <div id="ursh-shorten-request-form" class="i-has-action">
+        <input id="ursh-shorten-input" name="ursh-shorten-original-url" type="url" style="font-family:monospace;"
+               spellcheck="false" autocomplete="off" autocapitalize="off"
+               placeholder="{% trans %}Enter an Indico URL to shorten{% endtrans %}...">
```

Comment:
I'd keep the `...` inside the translated string (dunno if it's the same in all languages)